api/proxy/.env.example

```diff
@@ -0,0 +1,90 @@
+# This .env.example file only describes the mandatory flags that must be set when running the EigenDA proxy.
+# All flags that are optional and have sane default values are omitted for brevity.
+# For a full list of flags, please refer to the output of `eigenda-proxy --help`,
+# which is also available at https://github.com/Layr-Labs/eigenda/blob/master/api/proxy/docs/help_out.txt.
+
+# === Storage Configuration ===
+
+# The storage backends to enable. Options are [V1, V2]
+EIGENDA_PROXY_STORAGE_BACKENDS_TO_ENABLE=V1,V2
```

Comment:
Consider adding additional guidance here:

The contents of this field dictate whether the structures to support the given backend are instantiated or not.

Reasons to include both `V1` *and* `V2` include:
1. You're dispersing to one, but still want to support reads from the other
2. You want to change which backend you are dispersing to on the fly, i.e. you're migrating from V1 to V2. So everything would be going through V1 at first, but the necessary structures would exist to switch to V2 dispersals, without requiring a restart